src/consts/validation.js

```diff
@@ -13,7 +13,7 @@ const regex = {
 
 const enums = {
   APP_LANG_ENUM: ['en', 'ua'],
-  SPOKEN_LANG_ENUM: ['English', 'Ukrainian', 'Polish', 'German', 'French', 'Spanish', 'Arabic'],
+  SPOKEN_LANG_ENUM: ['English', 'Ukrainian', 'Polish', 'German', 'French', 'Spanish', 'Arabic', null],
```

Comment:
Maybe we shouldn't add it to the validation rule. By default it will be null, but after the user completes the stepper, he should only select the language, not null.

Reply:
I know ,but I add this , because without this I have error:   
{
    "status": 409,
    "code": "VALIDATION_ERROR",
    "message": "User validation failed: nativeLanguage: Native language can be either of these: English,Ukrainian,Polish,German,French,Spanish,Arabic"
}

Reply:
Well, I optimize my code . I don`t touch constants. All did in Users model